### PR TITLE
Update `Banner` component as per Figma design, and apply to `AllWorks…

### PR DESCRIPTION
**Pull request by miaozhaod**
…hopPage`

### CODE REVIEW

File: src/pages/AllWorshopsPage.jsx [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+import Banner from "../components/common/Banner";
 import Container from "../components/common/Container";
 import ProjectCoverCard from "../components/ProjectCoverCard";
 import { mockWorkshopsData } from "../mock/workshopsData";
 
 export default function AllWorkshopsPage() {
   return (
     <>
+      <Container wrapperVariant="banner">
+        <Banner>Expolore Workshops</Banner>
```
Comment (builtbyproxy) [inline src/pages/AllWorshopsPage.jsx:11]: Perhaps a spelling mistake?

File: src/components/common/Banner/index.jsx [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
   return (
-    <div className="banner">
-      <h1> {heading}</h1>
-      <div className={`${variant === "banner-tags" ? "banner-tags" : ""}`}>
-        {children}
-      </div>
+    <div
+      className={`banner ${variant === "center-align" ? "center-align" : ""}`}
+    >
+      <h1> {children}</h1>
```
Comment (builtbyproxy) [inline src/components/common/Banner/index.jsx:9]: Tiny nitpick but there's a space here between the `<h1>` tag and `{children}` call

File: src/pages/AllWorshopsPage.jsx (hunk 1 above)
Comment (miaozhaod) [reply src/pages/AllWorshopsPage.jsx:11]: oops, yes, fixed typo
Update has been pushed into the `develop` branch straightaway, I think I merged the branch to `develop` before I saw your comments 😂

File: src/components/common/Banner/index.jsx (hunk 2 above)
Comment (miaozhaod) [reply src/components/common/Banner/index.jsx:9]: yah, removed the space
Update has also been pushed into the `develop` branch straightaway

**Pull request merged**